frontend/src/views/DashboardPage/DashboardPage.tsx

```diff
@@ -746,8 +809,190 @@ export const DashboardPage = () => {
     ({ isReadDenied, isWriteDenied }) => !isReadDenied || !isWriteDenied
   );
 
+
+  const handleCheckedSecret = (secretObj: { _id: string, isChecked: string | boolean }) => {
+    const checkedSecretsClone = [...checkedSecrets]
+    const checkedSecretIndex = checkedSecretsClone.findIndex(secret => secret._id === secretObj._id)
+    if (secretObj.isChecked) {
+      checkedSecretsClone.push(secretObj)
+    } else {
+      checkedSecretsClone.splice(checkedSecretIndex, 1)
+    }
+    setCheckedSecrets(() => checkedSecretsClone)
+  }
+
+  const onMoveSecrets = async ($folderId: string, $checkedSecrets: { _id: string, isChecked: string | boolean }[]) => {
+    // eslint-disable-next-line no-alert
+    const confirm = window.confirm(`Are you sure you want to move  ${checkedSecrets.length > 1 ? "secrets" : "secret"}?`)
+    handlePopUpClose("moveSecrets")
+    if (confirm) {
+      const mappedCheckedSecrets = $checkedSecrets.map(checkedSecret => {
```

Comment:
Kindly follow the codebase naming convention and please move this dollar signs from all variables